components/atoms/Accordion2.js

```diff
@@ -0,0 +1,85 @@
+import propTypes from "prop-types";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
+
+export function Accordion2(props) {
+  const [openFlag, setOpenFlag] = useState(false);
+
+  function openSummary() {
+    setOpenFlag(!openFlag);
+  }
+
+  return (
+    <div className="flex mb-0 ">
+      <details
+        className={`accordion flex-1 flex-wrap border border-gray-300 rounded bg-white ((${props.mainClass})? ${props.mainClass} : '') `}
+      >
+        <summary onClick={() => openSummary()}>
+          <div className="w-12 min-w-12 relative left-4 pt-3 float-left ">
+            {openFlag ? (
+              <FontAwesomeIcon
+                icon={faChevronRight}
+                rotation={90}
+                size="1x"
+                className="text-custom-blue-reportButton"
+              />
+            ) : (
+              <FontAwesomeIcon
+                icon={faChevronRight}
+                size="1x"
+                className="text-custom-blue-reportButton"
+              />
+            )}
+          </div>
+
+          <div className="ml-8 pt-3 text-custom-blue-reportButton">
```

Comment:
Mentioned this before but on some of the accordions, the text is pretty close to touching the right edge, so maybe add a `pr-1` here (which forces it to the next line).
![Screenshot 2021-07-28 160413](https://user-images.githubusercontent.com/71025360/127388453-43155665-3925-4fd4-8221-f9957a53dfd0.png)


```suggestion
          <div className="ml-8 pt-3 pr-1 text-custom-blue-reportButton">
```